.jscsrc

```diff
@@ -0,0 +1,5 @@
+{
+  "preset": "airbnb",
+  "excludeFiles": ["!(app|lib)"],
```

Comment:
any reason why we're not excluding `vendor` here too? I'm not sure how many JS files are going to be in that directory but it might still help.